index.js

```diff
@@ -18,4 +19,6 @@ module.exports = function (file, args, options) {
 			resolve({stdout: stdout, stderr: stderr});
 		});
 	});
+	promise.child = child;
```

Comment:
No only `then` and `catch`, what you did at the moment. Setting another implementation as the global promise is a bad idea imo. And if they do that, they can still wrap it again into their own promise implementation if they want.